README.md

```diff
@@ -217,6 +217,43 @@ kubectl get pods -n nats-manager-system
 kubectl get -n kyma-system nats
 ```
 
+## E2E tests
```

Comment:
It's unclear to me whether the following instructions are alternatives to each other (then use a bullet list) or subsequent steps (then use a numbered list), or a mixture. Can you advise?

Reply:
@NHingerl  Thank you! You are right. I added numbers to make the consecutive nature of the steps clear.

Reply:
I see you added numbering for lines 240 ff 👍 
But what about the instructions in line 221, 227, an 232? Are they alternatives to each other?